packages/document/main-doc/docs/en/configure/app/output/source-map.mdx

```diff
@@ -17,7 +17,9 @@ type SourceMap = {
 - **Default:**
 
 ```ts
-const defaultSourceMap = {
+const defaultSourceMap =
```

Comment:
Should update types instead of updating default value

Reply:
😂